mod-circulation/src/main/resources/domain/mod-circulation/features/samples/item/item-entity-request.json

```diff
@@ -9,6 +9,6 @@
     "id": "#(permanentLoanTypeId)"
   },
   "status": {
-    "name": "Available"
+    "name": "#(itemStatusName)"
```

Comment:
Is this going to affect other tests using item-entity-request.json ?

Reply:
no effect to other scenarios, I gave default value in initData.feature